mlir/test/Dialect/Linalg/roundtrip.mlir

```diff
@@ -706,3 +706,21 @@ func.func @conv2d_channel_first_q_promote(%img: tensor<100x3x224x224xi8>, %filt:
 // CHECK-LABEL: func @conv2d_channel_first_q_promote(
 // CHECK:   %[[arg0:[a-zA-z0-9]*]]: tensor<100x3x224x224xi8>, %[[arg1:[a-zA-z0-9]*]]: tensor<64x3x5x5xi8>, %[[arg2:[a-zA-z0-9]*]]: i8, %[[arg3:[a-zA-z0-9]*]]: i8)
 // CHECK:         linalg.conv_2d_nchw_fchw_q {dilations = dense<1> : tensor<2xi64>, strides = dense<1> : tensor<2xi64>} ins(%[[arg0]], %[[arg1]], %[[arg2]], %[[arg3]] : tensor<100x3x224x224xi8>, tensor<64x3x5x5xi8>, i8, i8) outs(%{{.*}} : tensor<100x64x220x220xi32>) -> tensor<100x64x220x220xi32>
+
+// -----
+// Test that we can lower all the way to LLVM without crashing, don't check results here.
```

Comment:
I don't follow the comment.. Can you elaborate?

Reply:
Just wanted to check if mlir can lower to llvm ir.

Reply:
But this is a roundtrip test. It verifies that the `mlir-opt` can parse the IR and print the IR that matches the check. There are no LLVM IR involved.

Reply:
I'll check an appropriate file to put this test later; now I can't find one.

Reply:
What I meant is that it is legacy. Removing the comment is suitable to me. No need to check that